### PR TITLE
Add link redirecting to council/motions on bounty list when voting on bounty

### PR DESCRIPTION
**Pull request by ekowalsk**
image

### CODE REVIEW

File: packages/page-bounties/src/Voting/VotingLink.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+export default React.memo(styled(VotingLink)`
+  && .voting-link {
+    font-size: 0.85rem;
+    line-height: 0.5rem;
+    color: currentColor;
+    :hover {
+      color: currentColor;
+    }
+  }
```
Comment (jacogr) [inline packages/page-bounties/src/Voting/VotingLink.tsx:45]: The links should just be links (as it is everywhere atm) - the button-like appearance imho probably drives away from "open-in-new-window".

**Comment by ekowalsk** [comment]
I'm very, very sorry for the force-push again. There's no new changes

**Comment by jacogr** [comment]
Old habits die hard :)

**Review by jacogr — APPROVED** [review]
Looks good, has conflicts.

**Comment by ekowalsk** [comment]
Hi, I've added one more commit with improvements of css, would you like to have a look at it? Or it's ok if we merge it? I've also updated the image of how it looks like now

File: packages/page-bounties/src/Voting/VotingLink.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ function VotingLink (): React.ReactElement<string> {
   const { t } = useTranslation();
 
   return (
-    <div>
+    <div style={{ margin: ' 0 1rem 0 0' }}>
```
Comment (jacogr) [inline packages/page-bounties/src/Voting/VotingLink.tsx:13]: Would really prefer not to do this. This is a nightmare to maintain.

Comment (ekowalsk) [reply packages/page-bounties/src/Voting/VotingLink.tsx]: ok, fixed :)

**Pull request merged**